code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -37,10 +37,16 @@ emp_act
 		return 2
 
 	if(mind?.martial_art?.deflection_chance) //Some martial arts users can deflect projectiles!
-		if(!lying && !HAS_TRAIT(src, TRAIT_HULK) && prob(mind.martial_art.deflection_chance)) //But only if they're not lying down, and hulks can't do it
+		if(!lying && !HAS_TRAIT(src, TRAIT_HULK) && mind?.martial_art?.try_deflect(src)) //But only if they're not lying down, and hulks can't do it
```

Comment:
You shouldn't need the extra checks here, since they're already done on the line above.
```suggestion
		if(!lying && !HAS_TRAIT(src, TRAIT_HULK) && mind.martial_art.try_deflect(src)) //But only if they're not lying down, and hulks can't do it
```